vars/pipelineSnapshotTesting.groovy

```diff
@@ -5,30 +5,54 @@
 import org.jenkinsci.plugins.workflow.steps.FlowInterruptedException
 import org.jenkinsci.plugins.pipeline.modeldefinition.Utils
 
-void call(REMOTE_SERVER="n01.d.vega.xyz") {
+void call(Map config=[:]) {
+
+    String vegaNetwork = config.network ?: 'devnet1'
+    Map<String, List<String>> serversByNetwork = [
+        'devnet1': (1..4).collect { id ->  "n0${id}.d.vega.xyz" },
```

Comment:
nit: did you know you can write: `'devnet1': (1..4).collect { "n0${it}.d.vega.xyz" }`